pydoctor/model.py

```diff
@@ -267,9 +267,11 @@ def _handle_reparenting_post(self) -> None:
         self.system.allobjects[self.fullName()] = self
         for o in self.contents.values():
             o._handle_reparenting_post()
-
     def _localNameToFullName(self, name: str) -> str:
         raise NotImplementedError(self._localNameToFullName)
+    
+    def isNameDefined(self, name:str) -> bool:
+        raise NotImplementedError(self.isNameDefined)
```

Comment:
Should this become an `ABC` as well, given that this is really an `abstractmethod`?